test/dotnet.Tests/PackagedCommandTests.cs

```diff
@@ -91,6 +91,82 @@ public void TestProjectToolIsAvailableThroughDriver(string appName, bool useCurr
                      .And.Pass();
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void IfPreviousVersionOfSharedFrameworkIsNotInstalled_ToolsTargetingItFail(bool toolPrefersCLIRuntime)
+        {
+            var testInstance = TestAssets.Get("AppWithToolDependency")
+                .CreateInstance(identifier: toolPrefersCLIRuntime ? "preferCLIRuntime" : "")
+                .WithSourceFiles()
+                .WithNuGetConfig(new RepoDirectoriesProvider().TestPackages);
+
+            testInstance = testInstance.WithProjectChanges(project =>
+            {
+                var ns = project.Root.Name.Namespace;
+
+                var toolReference = project.Descendants(ns + "DotNetCliToolReference")
+                    .Where(tr => tr.Attribute("Include").Value == "dotnet-portable")
+                    .Single();
+
+                toolReference.Attribute("Include").Value =
+                    toolPrefersCLIRuntime ? "dotnet-portable-v1-prefercli" : "dotnet-portable-v1";
+            });
+
+            testInstance = testInstance.WithRestoreFiles();
+
+            new BuildCommand()
+                .WithProjectDirectory(testInstance.Root)
+                .Execute()
+                .Should().Pass();
+
+            new GenericCommand(toolPrefersCLIRuntime ? "portable-v1-prefercli" : "portable-v1")
+                .WithWorkingDirectory(testInstance.Root)
+                .WithEnvironmentVariable("DOTNET_MULTILEVEL_LOOKUP", "0")
```

Comment:
In 2.0, the dotnet host will look in your user folder and in Program files for shared frameworks, in addition to the path where dotnet.exe is.  These tests are supposed to test that tools targeting 1.x fail to run if the 1.x shared framework isn't installed, so this environment variable stops it from resolving to the shared frameworks that you might have installed globally on your machine.

Perhaps we should set this environment variable all the time for the CLI.  @livarcocc